system/ee/ExpressionEngine/View/_shared/header.php

```diff
@@ -3,7 +3,7 @@
     <head>
         <?=ee()->view->head_title($cp_page_title)?>
         <meta http-equiv="content-type" content="text/html; charset=utf-8">
-        <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"  name="viewport">
```

Comment:
Just making sure there will be no undesired consequences on different screen resolutions if we remove this (e.g. in modals, slide-outs, jump menu behaviour etc)

Reply:
`Maximum-scale is an optional attribute for the viewport, and it defines the maximum zoom that website users are able to do. Setting a value for the 'maximum-scale' attribute could prevent the user from scaling properly, meaning that users may not be able to zoom in/out on mobile devices, and may cause accessibility issues.`
`The user-scalable attribute defines whether users can scale (zoom) the page content on mobile. By default, scaling is on, but if user-scalable is set to '0' or 'no' then this will prevent users from zooming, which negatively affects user experience (in most cases).`
<img width="835" alt="Screenshot 2023-08-07 at 13 39 44" src="https://github.com/ExpressionEngine/ExpressionEngine/assets/23382425/006f186e-bbff-42d2-acdb-86b7be28fef3">
The recommended, 'safe' viewport setting is which we have on site:
`<meta name="viewport" content="width=device-width, initial-scale=1">`

Reply:
@Yulyaswan this does break things :(
On File Manager page on iPhone 12 (Chrome simulator) in horizontal mode the main sidebar is not being automatically collapsed, making it nearly impossible to use.